README.md

```diff
@@ -78,6 +78,15 @@ Follow the given instructions for Login into the app.
 
 To run the tests run: `python manage.py test`.
 
+## Documentation
```

Comment:
@Amulya-coder I think it would be more appropriate to change the title from "Documentation" to "About"

Reply:
@khushishikhu we work on the documentation section of this repo so only about doesn't make much sense, I had taken a      
reference from this: https://github.com/anitab-org/mentorship-backend#documentation please check it.